packages/js-client/test/mocks/aragon-sdk-ipfs.ts

```diff
@@ -0,0 +1,23 @@
+export const mockedIPFSClient = {
```

Comment:
I would add some comment here, explaining to devs that these are default values and also how to add their own desired ones ad-hoc.

Reply:
👍